fs/config/configflags/configflags.go

```diff
@@ -137,6 +137,8 @@ func AddFlags(ci *fs.ConfigInfo, flagSet *pflag.FlagSet) {
 	flags.BoolVarP(flagSet, &ci.DisableHTTP2, "disable-http2", "", ci.DisableHTTP2, "Disable HTTP/2 in the global transport")
 	flags.BoolVarP(flagSet, &ci.HumanReadable, "human-readable", "", ci.HumanReadable, "Print numbers in a human-readable format, sizes with suffix Ki|Mi|Gi|Ti|Pi")
 	flags.DurationVarP(flagSet, &ci.KvLockTime, "kv-lock-time", "", ci.KvLockTime, "Maximum time to keep key-value database locked by process")
+	flags.BoolVarP(flagSet, &ci.PreferIPv4, "prefer-ipv4", "", ci.PreferIPv4, "Force rclone to prefer IPv4 connections for all backends")
```

Comment:
This should probably be `Force rclone to prefer IPv4 connections` - it isn't just for backends.

This would traditionally have the short flag `-4` - which might be worth adding?